vulkan/vulkan_window.cc

```diff
@@ -109,7 +108,13 @@ bool VulkanWindow::CreateSkiaGrContext() {
     return false;
   }
 
+// This is just to make lint happy: MakeVulkan isn't defined unless SK_VULKAN is
+// defined, and lint doesn't define SK_VULKAN.
+#ifdef SK_VULKAN
```

Comment:
Can we just use NOLINT?  I'd prefer that.

Reply:
So would I, but no, without those `#ifdef`s, it's not just a lint error, it's a syntax error, since `MakeVulkan` isn't defined at all if `SK_VULKAN` isn't.